vaadin-date-time-picker-flow-integration-tests/pom.xml

```diff
@@ -56,6 +56,7 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>flow-test-util</artifactId>
+            <scope>compile</scope>
```

Comment:
What does this do? It seems date picker repo doesn't have this, but time picker does.